### PR TITLE
Binding Parser grammar update

### PR DESCRIPTION
**Pull request by sahankj2000**
### Description
- Issue: [#1134](https://github.com/nosqlbench/nosqlbench/issues/1134)
- Invalid construction of bindings is silently discarded, but an error should be thrown
- The case `Add(10) Add(20)` should be working (space should work in ileu of a semicolon)

### Comparison
- Space instead of semicolon: `Add(10) Add(20)`
    ```
    # before
    $ java -jar nb5.jar run workload=iot.yaml tags=block:main-read cycles=10 driver=stdout
    10
    11
    ...
    
    # new
    $ java -jar nb5.jar run workload=iot.yaml tags=block:main-read cycles=10 driver=stdout
    30
    31
    ...
    ```
- Invalid syntax: `Add(10); - Add(20)`
  ```
  # before
  $ java -jar nb5.jar run workload=iot.yaml tags=block:main-read cycles=10 driver=stdout
  line 1:9 token recognition error at: '- '
  30
  31
  ...
  
  # new
  $ java -jar nb5.jar run workload=iot.yaml tags=block:main-read cycles=10 driver=stdout
  line 1:9 token recognition error at: '- '
      6164 WARN  [main] VirtDataDSL  Error while parsing flow:LexerNoViableAltException('-')
      6165 ERROR [main] NBINVOKE     error   [CMD_run {container="default",instance="default",jobname="nosqlbench",node="192.168.1.34",session="nYmU6Fq",step="run"}]: (stack trace follows): CMD_run {container="default",instance="default",jobname="nosqlbench",node="192.168.1.34",session="nYmU6Fq",step="run"}: java.lang.RuntimeException: Error while parsing binding specification 'Add(10); - Add(20)': java.lang.RuntimeException: LexerNoViableAltException('-')
  java.lang.RuntimeException: Error while parsing binding specification 'Add(10); - Add(20)': java.lang.RuntimeException: LexerNoViableAltException('-')
	  at io.nosqlbench.virtdata.core.bindings.VirtData.getOptionalMapper(VirtData.java:102)
	  at io.nosqlbench.virtdata.core.templates.StringCompositor.lambda$new$1(StringCompositor.java:52)
	  at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	  at io.nosqlbench.virtdata.core.templates.StringCompositor.<init>(StringCompositor.java:51)
	  at io.nosqlbench.virtdata.core.templates.StringCompositor.<init>(StringCompositor.java:75)
	  at io.nosqlbench.virtdata.core.templates.StringBindings.<init>(StringBindings.java:49)
	  at io.nosqlbench.virtdata.core.templates.StringBindings.<init>(StringBindings.java:45)
  ...
  ```

### CODE REVIEW

File: nb-virtdata/virtdata-lang/src/main/java/io/nosqlbench/virtdata/lang/parser/VirtDataDSL.java [hunk 1]
```diff
@@ -67,6 +62,18 @@ public static VirtDataDSL.ParseResult parse(String input) {
         }
     }
 
+    private static VirtDataParser getVirtDataParser(CodePointCharStream cstream) {
```
Comment (Prodyte) [inline nb-virtdata/virtdata-lang/src/main/java/io/nosqlbench/virtdata/lang/parser/VirtDataDSL.java:70]: are these changes required?

Comment (sahankj2000) [reply nb-virtdata/virtdata-lang/src/main/java/io/nosqlbench/virtdata/lang/parser/VirtDataDSL.java:70]: Only added the error listener here to throw error when there is some syntax error otherwise if there are syntax errors there will be a line printed for it, but execution will continue.

When I added this IntelliJ was showing a yellow line saying it can be extracted into a method, so did that.
Let me know if you think we don't need this extra method, I can revert it!!!

Comment (dave2wave) [reply nb-virtdata/virtdata-lang/src/main/java/io/nosqlbench/virtdata/lang/parser/VirtDataDSL.java:70]: I think that the ".*" form of includes is not a good practice. Otherwise I think that shooky would approve.
